brem/__init__.py

```diff
@@ -29,6 +29,7 @@
     from .brem import BasicRemoteExecutionManager
     from .brem import RemoteRunControl
     from .brem import RemoteRunControlSignals
+    from .AsyncCopyOverSSH import AsyncCopyOverSSH, RemoteAsyncCopyOverSSHSignals
     
 
 except ImportError:
```

Comment:
If we change to
```
except ImportError as e:
    raise ImportError('error importing brem: ', e)
```

Then we will see the details of the error in future

Reply:
What would it change?